src/dec128.rs

```diff
@@ -456,6 +458,16 @@ impl d128 {
         CTX.with(|ctx| f(&mut ctx.borrow_mut()))
     }
 
+    /// Creates from raw bytes
```

Comment:
I think a better interface is to say that from_bytes() is machine dependent. The user must handle endianess manually if it cares to do so. As such the documentation should be:

```rust
/// Creates a d128 from raw bytes. Endianess is host dependent.
pub unsafe fn from_raw_bytes(bytes: [u8; 16]) -> d128 ...
```